packages/flex-plugin-scripts/src/scripts/start.ts

```diff
@@ -48,24 +75,21 @@ const start = async (...args: string[]) => {
       }
       throw err;
     });
-  }
-  if (args[0] === 'plugin') {
+  } if (args[0] === 'plugin') {
     type = WebpackType.JavaScript;
-  }
-
-  const index = args.indexOf('--name');
-
-  if (index !== -1) {
-    const config = readJsonFile<CLIFlexConfiguration>(paths().cli.pluginsJsonPath);
-    const plugin = config.plugins.find((p) => p.name === args[index + 1]);
+    const plugin = userInputPlugins.filter(p => !p.remote)[0];
```

Comment:
`const plugin = userInputPlugins.find(p => !p.remote);` instead 

Also no guarantee `plugin` is actually found - you need to do a check `if (!plugin)`

Reply:
I think an `if (plugin)` check instead that does this might be more useful:
`const config = readPluginsJson();
      const pluginFind = config.plugins.find((p) => p.name === plugin.name);

      if (pluginFind) {
        _updatePluginPort(port, pluginFind.name);
        setCwd(pluginFind.dir);
      }`
So that if there are no local plugins/there are just remote plugins, the remote one is still able to run

Reply:
`plugin` could be undefined which mean `config.plugins.find((p) => p.name === plugin.name);` would throw an NPE (you are doing `plugin.name`) inside it.